RELEASE_NOTES.md

```diff
@@ -1,5 +1,23 @@
 # Release Notes
 
+## stable-1.20-4.4.0, v4.4.0
+
+### kube-oidc-proxy
+  - * bump kube-oidc-proxy to 0.3.0
+  #1420 (@mesosphere-mergebot)
+
+### nvidia
+  - Fixes Nvidia GPU platform service placement constraints.
+  #1417 (@mesosphere-mergebot)
+
+  - updates Nvidia DCGM exporter to 2.2.9
+  #1412 (@mesosphere-mergebot)
+
+### prometheus
+  - [prometheus] Prometheus addon upgrades now properly upgrade the alertmanager CRD (COPS-6842)
+  #1330 (@mesosphere-mergebot)
```

Comment:
```suggestion
  - Alertmanager CRD is now properly upgraded with Prometheus addon (COPS-6842)
  #1330 (@mesosphere-mergebot)

  - Adds concurrency policy Replace configuration to the home dashboard CronJob to avoid potentially creating an unbound number of pods (COPS-7105)
  #1399 (@mesosphere-mergebot)
```